xml/ha_crmreport_passl.xml

```diff
@@ -336,7 +328,7 @@
     </para>
 <screen>Host_Alias	CLUSTER = &node1;,&node2;,&node3; <co xml:id="ha.sudoers.host.alias"/>
 User_Alias HA = hareport <co xml:id="ha.sudoers.user.alias"/>
-Cmnd_Alias HA_ALLOWED = /bin/su ,/usr/sbin/hb_report,/usr/sbin/crm_report <co xml:id="ha.sudoers.cmd.alias"/>
+Cmnd_Alias HA_ALLOWED = /bin/su,/usr/sbin/hb_report,/usr/sbin/crm_report <co xml:id="ha.sudoers.cmd.alias"/>
```

Comment:
Hmm, I think this then ought to be 

`Cmnd_Alias HA_ALLOWED = /bin/su ,/usr/sbin/crm report *`